### ISSUE TITLE
Document that icons should have screen reader accessible text when they're not redundant or decorative

### ISSUE THREAD

**Issue by missmatsuko** [adobe/spectrum-css#418]
## Description

[Iconography](https://spectrum.adobe.com/page/iconography/) page doesn't mention the importance of exposing screen reader accessible text when icons are used to provide meaning.

[Action button options](https://spectrum.adobe.com/page/action-button/#Options) say "Action buttons should have a label whenever possible for accessibility. If a label isn’t present, it must include an icon and it becomes an icon-only action button." 

It doesn't seem clear to me that an icon-only action button should still have a label, even if it's not a visible label. If an action button's content is an icon without any text, screen readers users won't know what it's for.

**Comment by GarthDB** [comment]
We still need to do this.

**Comment by misterbrownlee** [comment]
This is a problem with the design website, so we will forward this to the design backlog.  Closing this.

**Comment by missmatsuko** [comment]
> This is a problem with the design website, so we will forward this to the design backlog. Closing this.

This is a documentation issue. There are many similar documentation issues in this repo and even a documentation label. Are you moving all of them to the design backlog or just this one? If it's just this one... why?
